skimage/transform/_geometric.py

```diff
@@ -970,8 +1047,64 @@ def inverse(self, coords):
         return out
 
 
+def _euler_rotation(axis, angle):
+    """Produce a single-axis Euler rotation matrix.
+
+    Parameters
+    ----------
+    axis : int in {0, 1, 2}
+        The axis of rotation.
+    angle : float
+        The angle of rotation in radians.
+
+    Returns
+    -------
+    Ri : array of float, shape (3, 3)
+        The rotation matrix along axis `axis`.
+    """
+    i = axis
+    s, c = np.sin(angle), np.cos(angle)
+    R2 = np.array([[          c, (-1)**(i+1) * s],
+                   [(-1)**i * s,               c]])
```

Comment:
```suggestion
    s, c = ((-1) ** i) * np.sin(angle), np.cos(angle)
    R2 = np.array([[c, -s],
                   [s, c]])
```

Reply:
done, thanks for the suggestion @rfezzani, this is indeed clearer. =)